packages/core/root/core-root-browser-internal/src/kbn_bootstrap.ts

```diff
@@ -47,4 +47,12 @@ export async function __kbnBootstrap__() {
 
   const start = await coreSystem.start();
   await apmSystem.start(start);
+
+  window.addEventListener('unload', () => {
```

Comment:
With all the inconsistency around `unload` I just wanted to ask FMI: it seems like unload can be called by browsers trying to free up resources etc, does this mean when the user returns to the "unloaded" page we'll have the chance to call `setup`, `start` again? Will it generally act like a page reload in that case?